crates/ruff/src/message/mod.rs

```diff
@@ -132,6 +132,7 @@ mod tests {
     pub(super) fn create_messages() -> Vec<Message> {
         let fib = r#"import os
 
+
```

Comment:
I added an extra empty line here to demonstrate that it truncates trailing empty lines. I had to change the line number of the `x` diagnostic because of that and it propagated to all snapshots.